terraform/7-rds.tf

```diff
@@ -0,0 +1,94 @@
+module "db" {
+  source  = "terraform-aws-modules/rds/aws"
+
+  identifier = "libera-postgresql"
+
+  # All available versions: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html#PostgreSQL.Concepts
+  engine               = "postgres"
+  engine_version       = "14"
+  family               = "postgres14" # DB parameter group
+  major_engine_version = "14"         # DB option group
+  instance_class       = "db.t4g.large"
+
+  allocated_storage     = 20
+  max_allocated_storage = 100
+
+  create_db_option_group = true
+  create_db_parameter_group = true
+
+
+  # NOTE: Do NOT use 'user' as the value for 'username' as it throws:
+  # "Error creating DB Instance: InvalidParameterValue: MasterUsername
+  # user cannot be used as it is a reserved word used by the engine"
+  db_name  = "liberadbname"
+  username = "liberadbusername"
+  port     = 5432
+  # TODO Ver el tema de la password
+
+  multi_az               = true
+  db_subnet_group_name   = module.vpc.database_subnet_group
+  vpc_security_group_ids = [module.security_group.security_group_id]
+
+  maintenance_window              = "Mon:00:00-Mon:03:00"
+  backup_window                   = "03:00-06:00"
+  enabled_cloudwatch_logs_exports = ["postgresql", "upgrade"]
+  create_cloudwatch_log_group     = true
+
+  backup_retention_period = 7
+  skip_final_snapshot     = true  # TODO Switch
```

Comment:
porque ese TODO Switch mano?

Reply:
Because during testing, it would be a mess to have that turned on, I just added a new comment explaining that.